src/develop.js

```diff
@@ -0,0 +1,3 @@
+let a = 'Hope I got it right';
+console.log (a);
```

Comment:
As I said, this will not work
Remove unnesesary space
```suggestion
console.log(a);
```